packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.doc.tsx

```diff
@@ -50,7 +50,7 @@ export const DetailsListPageProps: IDocPageProps = {
   componentStatus: DetailsListStatus,
   examples: [
     {
-      title: 'Document DetailsList with 500 items, sorting, filtering, marquee selection, justified columns',
```

Comment:
I mainly changed the wording because I don't think the phrase "Document DetailsList" makes sense, but I'm open to other suggestions of how to indicate that the list contains documents